server/mediapublic/models.py

```diff
@@ -451,8 +451,8 @@ def to_dict(self):
             facebook=self.facebook,
             instagram=self.instagram,
             periscope=self.periscope,
-            user_id=self.user_id,
-            organization_id=self.organization_id,
+            user_id=str(self.user_id),
```

Comment:
`user_id` and `organization_id` are both UUID.  The json serializer can't take them as UUID.  Are you saying that something has to change here?